serverless.yaml

```diff
@@ -43,6 +43,15 @@ plugins:
 
 custom:
   stages:
+    personal:
```

Comment:
Can we just use the development stage instead? Should be able to change profile to `profile: ${{env:AWS_PROFILE}}`

Reply:
oh good idea, i will change that